### ISSUE TITLE
Skip the break function

### ISSUE THREAD

**Issue by Hexodus** [schmich/marinara#67]
Sometimes it would be convenient to be able to skip a break. When I'm in the flow state I want to use it for working further.

**Comment by schmich** [comment]
Makes sense. I can add an option to the desktop notification and a link on the expiration page to skip the break and immediately start the next focus session.

Thanks for the feedback. I'll get this into the next release.

**Comment by blipinsk** [comment]
@schmich did you end up adding this?
